src/main/java/de/tum/cit/aet/artemis/exam/domain/event/ExamLiveEvent.java

```diff
@@ -38,7 +38,7 @@
 public abstract class ExamLiveEvent extends DomainObject {
 
     @CreatedBy
-    @Column(name = "created_by", nullable = false, length = 50, updatable = false)
+    @Column(name = "created_by", nullable = false, length = 70, updatable = false)
```

Comment:
I assume `createdBy` contains the first and last name of the user who created the exam live event.
According to the database scheme, both of them can contain up to 50 characters.

https://github.com/ls1intum/Artemis/blob/5b840700dd57f6d383fec2f936cc8955b434fdc4/src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml#L996-L997

Therefore, I suggest to align this with the jhi_user table and use a maximum length of 100.